src/openzaak/components/documenten/widgets.py

```diff
@@ -3,5 +3,7 @@
 
 class PrivateFileWidget(_PrivateFileWidget):
     def get_display_value(self, value):
-        instance = value.instance
-        return instance.bestandsnaam or super().get_display_value(value)
+        instance = getattr(value, "instance", None)
+        return getattr(instance, "bestandsnaam", None) or super().get_display_value(
+            value
+        )
```

Comment:
```suggestion
        if not value:
            return super().get_display_value(value)

        instance = value.instance
        return instance.bestandsnaam or super().get_display_value(value)
```

Isn't this cleaner, because `value` is `None` when adding a new document?

Reply:
yes you're right. separates the two cases more clearly